DSA/1.1 Arrays/3. Range Queries/1.1.3.1_Arrays: Range_Queries_(Sum_and_Product).cpp

```diff
@@ -0,0 +1,126 @@
+// Solution for Array Range sum and Range product queries 
+
+
+#include <bits/stdc++.h>
+
+using namespace std;
+
+ 
+ void preSum(int array[], int n, int prefixSum[])
+    {
+        prefixSum[0] = array[0];
+        
+        for (int i = 1; i < n; i++)
+        {
+            prefixSum[i] = array[i] + prefixSum[i - 1];
+        }
+    }
+ 
+// Solution to Pre-Compute the sum for every query 
```

Comment:
instead of commenting , use simple text with highlighter and add theory at starting.